src/purify.ts

```diff
@@ -9,32 +9,32 @@ export function purify(content: string) {
   const newContent = content
     /* wrap TS 2.2 enums w/ an IIFE */
     .replace(
-      /var (\S+) = \{\};\n(\1\.(\S+) = \d+;\n)+\1\[\1\.(\S+)\] = "\4";\n(\1\[\1\.(\S+)\] = "\S+";\n*)+/mg,
+      /var (\S+) = \{\};\r?\n(\1\.(\S+) = \d+;\r?\n)+\1\[\1\.(\S+)\] = "\4";\r?\n(\1\[\1\.(\S+)\] = "\S+";\r?\n*)+/mg,
       'var $1 = /*@__PURE__*/(function() {\n$&; return $1;})();\n',
     )
     /* wrap TS 2.3 enums w/ an IIFE */
     .replace(
       // tslint:disable-next-line:max-line-length
-      /var (\S+);(\/\*@__PURE__\*\/)*\n\(function \(\1\) \{\s+(\1\[\1\["(\S+)"\] = 0\] = "\4";(\s+\1\[\1\["\S+"\] = \d\] = "\S+";)*\n)\}\)\(\1 \|\| \(\1 = \{\}\)\);/mg,
+      /var (\S+);(\/\*@__PURE__\*\/)*\r?\n\(function \(\1\) \{\s+(\1\[\1\["(\S+)"\] = 0\] = "\4";(\s+\1\[\1\["\S+"\] = \d\] = "\S+";)*\r?\n)\}\)\(\1 \|\| \(\1 = \{\}\)\);/mg,
       'var $1 = /*@__PURE__*/(function() {\n    var $1 = {};\n    $3    return $1;\n})();',
     )
     /* Prefix safe imports with pure */
     .replace(
-      new RegExp(`(_(${pureImportMatches})__ = )(__webpack_require__\\("\\S+"\\));`, 'mg'),
+      new RegExp(`(_(${pureImportMatches})__ = )(__webpack_require__\\(\\S+\\);)`, 'mg'),
       '$1/*@__PURE__*/$3',
     )
     /* Prefix default safe imports with pure */
     .replace(
-      new RegExp(`(_(${pureImportMatches})___default = )(__webpack_require__\\.\\w\\(\\S+\\));`, 'mg'),
+      new RegExp(`(_(${pureImportMatches})___default = )(__webpack_require__\\.\\w\\(\\S+\\);)`, 'mg'),
       '$1/*@__PURE__*/$3',
     )
     /* Prefix CCF and CMF statements */
     .replace(
-      /__WEBPACK_IMPORTED_MODULE_0__angular_core__\["_\w+" \/\* (ɵccf|ɵcmf) \*\/\]\(/mg,
```

Comment:
Yeah, hard-coded 0 is definitely wrong. We ended up fixing this on the purify end in 0.0.24 (after the split)